code/obj/item.dm

```diff
@@ -307,6 +308,11 @@
 	if (isnull(initial(src.health))) // if not overridden
 		src.health = get_initial_item_health(src.type)
 	..()
+	if(prob(src.mimic_chance))
+		SPAWN(10 SECONDS)
+			var/mob/living/critter/mimic/replacer = new(get_turf(src.loc))
+			replacer.disguise_as(src)
+			qdel(src)
```

Comment:
maybe split this out into a proc on `/item` (or `/obj`)? Mostly so that admins can turn arbitrary stuff into mimics via proccall 😉